test/unit/core/type_list_test.cpp

```diff
@@ -19,3 +20,167 @@ TEST(type_list, basic)
 
     EXPECT_TRUE((std::is_same_v<meta::at_c<t, 1>, char>));
 }
+
+// ----------------------------------------------------------------------------
+// pack tests
+// ----------------------------------------------------------------------------
+
+TEST(pack_traits, size)
+{
+    EXPECT_EQ(( pack_traits::size<int, bool &, double const> ),
+                3u );
+}
+
+TEST(pack_traits, at)
+{
+    EXPECT_TRUE((std::is_same_v<pack_traits::at<2, int, bool &, double const, long, float>,
+                                double const>));
+    EXPECT_TRUE((std::is_same_v<pack_traits::at<-2, int, bool &, double const, long, float>,
+                                long>));
+}
+
+TEST(pack_traits, front)
+{
+    EXPECT_TRUE((std::is_same_v<pack_traits::front<int, bool &, double const, long, float>,
+                                int>));
+}
+
+TEST(pack_traits, back)
+{
+    EXPECT_TRUE((std::is_same_v<pack_traits::back<int, bool &, double const, long, float>,
+                                float>));
+}
+
+TEST(pack_traits, concat)
+{
+    EXPECT_TRUE((std::is_same_v<pack_traits::concat<int, bool &, double const, long, float>,
```

Comment:
I think we really don't need this. There is no added functionality by this.

Reply:
It's just an alias :man_shrugging:

Reply:
but as I said above an alias that does allow more functionality than it's type list counterpart